database/models/Product.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+const productSchema = new mongoose.Schema({
+    id: Number,
+    productName:String,
+    productPrice: Number,
+    productImageURL: String,
+    dateCreated: Date,
+    dateModified: Date,
+    categoryName: String
+});
+
+const Product = mongoose.model('product', productSchema);
+
+const getProducts = (name) => {
+    //console.log('getProducts is called');
+     return Product.find({ categoryName: "Sleek"}).sort({dateCreated : +1}).limit(16);   
```

Comment:
I hardcoded the categoryName for now...but that should come from the client as "req.body" once user selects the product